pkg/daemons/executor/etcd.go

```diff
@@ -27,8 +33,18 @@ func (e Embedded) ETCD(args ETCDConfig) error {
 
 	go func() {
 		select {
+		case err := <-etcd.Server.ErrNotify():
+			if strings.Contains(err.Error(), etcdserver.ErrMemberRemoved.Error()) {
+				tombstoneFile := filepath.Join(args.DataDir, "tombstone")
+				if err := ioutil.WriteFile(tombstoneFile, []byte{}, 0600); err != nil {
+					logrus.Fatal("failed to write tombstone file to %s", tombstoneFile)
+				}
+				logrus.Infof("this node has been removed from the cluster please restart %s to rejoin the cluster", version.Program)
```

Comment:
What is the actual behavior after seeing this message? Is k3s now stopped? Is it running but non-functional?

Reply:
running but not functional to imitate the behavior of external db